src/main/steps/urls.ts

```diff
@@ -72,6 +72,9 @@ export const DOCUMENT_MANAGER: PageLink = '/document-manager';
 export const SENT_TO_APPLICANT2_FOR_REVIEW: PageLink = '/application-sent-for-review';
 export const PAY_YOUR_FEE: PageLink = '/pay-your-fee';
 
+export const ENTER_YOUR_ACCESS_CODE: PageLink = '/enter-your-access-code';
```

Comment:
Does there need to be anything in the `src/main/modules/state-redirect/index.ts` to make sure that applicant 2 is redirected to this page when they register/login for the first time or is this taken care of by the email?

Reply:
@lara-mpt is currently working on the link that the second applicant will use and that will redirect them to the enter-your-access-code page